src/main/java/org/terasology/metalrenegades/ai/actions/CheckDaytimeAction.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.metalrenegades.ai.actions;
+
+import org.terasology.logic.behavior.BehaviorAction;
+import org.terasology.logic.behavior.core.Actor;
+import org.terasology.logic.behavior.core.BaseAction;
+import org.terasology.logic.behavior.core.BehaviorState;
+import org.terasology.metalrenegades.ai.system.DaylightTrackerSystem;
+import org.terasology.registry.CoreRegistry;
+import org.terasology.registry.In;
+
+/**
+ * Behavior node that checks the current daytime status, succeeds at daytime and fails at nighttime.
+ */
+@BehaviorAction(name = "check_daytime")
+public class CheckDaytimeAction extends BaseAction {
```

Comment:
Just a small note - you almost always invert this node in the behaviour trees, what about making it `"check_nighttime"` instead?

Reply:
I didn't expect to have to use so many inversions in the behaviors, looking at it now it would be more effective to change it to nighttime instead, I'll change it now 👍